app/controllers/api/guides_controller.rb

```diff
@@ -33,5 +38,26 @@ def destroy
                                           user: current_user)
       respond_with_mutation(:no_content)
     end
+
+    private
+
+    def check_if_crop_exists(params)
+      unless params[:crop_id].present?
+        puts params[:crop_name]
+        crops = Crop.search params[:crop_name], fields: [{name: :exact}]
+        puts 'search found', crops.to_json
+        if crops.count == 0
+          crop = Crop.new({name: params[:crop_name],
+                    common_names: [params[:crop_name],]})
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Space inside } missing.